### PR TITLE
Fix dropdown height problem

### PR DESCRIPTION
**Pull request by theomoura**
#### What is the purpose of this pull request?
As title says

#### Screenshots or example usage
image

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/components/Dropdown/index.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ class Dropdown extends Component {
 
     let classes = 'bg-transparent bn w-100 h-100 '
     let containerClasses = `${isInline ? '' : 'bw1'} br2 relative `
-    let selectClasses = 'o-0 absolute top-0 left-0 w-100 bottom-00 '
+    let selectClasses = 'o-0 absolute top-0 left-0 w-100 h-100 bottom-00 '
```
Comment (lbebber) [inline react/components/Dropdown/index.js]: I think the proper solution here would be to fix the `bottom-00` class—the correct name is `bottom-0`

Comment (ohmyguigs) [reply react/components/Dropdown/index.js]: This problem occurs only in Windows Chrome and MS Edge.`h-100` is not necessary, just fixing the `bottom-00` typo for `bottom-0` fixes it.

```suggestion
    let selectClasses = 'o-0 absolute top-0 left-0 w-100 bottom-0 '
```

Comment (theomoura) [reply react/components/Dropdown/index.js]: This seems to resolve the problem. Thx @lbebber @ohmyguigs

**Comment by ohmyguigs** [comment]
I'm gonna close this PR since we fixed this in 8.x and 9.x already on vtex/styleguide/pull/520

**Pull request closed without merging**